addons/sale/static/src/js/variant_mixin.js

```diff
@@ -363,7 +363,6 @@ var VariantMixin = {
         $parent
             .find('option, input, label')
             .removeClass('css_not_available')
-            .attr('title', '')
```

Comment:
I believe this is done because the '_disableInput' method sets a "Not available" title on the element (see line 451).
To avoid having the title still say unavailable when you check other options making this one available again.

Maybe some kind of check could be made to avoid that?
Like only removing the title if it matches the exact translation here under?
Seems a bit dirty, maybe you will think of a better solution.

Reply:
Big apologize for the delay and Thanks for the review!
I updated the fix (set default value as title if element have a `value_name` data attribute), what do you think?

Reply:
@awa-odoo  Big apologize for the delay and Thanks for the review!
I updated the fix (set default value as title if element have a `value_name` data attribute), what do you think?

Reply:
Looks better indeed.
However, using jQuery each would make things easier IMO.

```js
$parent.find('option, input, label').each(function () {
    $(this).removeClass('css_not_available')
        .data('excluded-by', '')
        .attr('title', $(this).data('value_name') || '');
})
```

Otherwise if it's tested and works properly, LGTM!

Reply:
Thanks, cleaner like always :wink: 
I tested again and I think you can r+ :+1:

Reply:
My r+ rights are limited to "Thibault is on vacation and some strange fix in our pipe cannot wait for his return" :D 

As this is related to a task / request from your team, I would prefer if you could ask your equivalent of my Thibault to r+ it?

Reply:
Alright thanks!